frontend/src/app/components/dashboard/db-table-view/db-table-view.component.css

```
@@ -401,12 +1110,11 @@
 		display: grid;
 		grid-template-columns: subgrid;
 		grid-column: 1 / 3;
-		grid-gap: 12px 28px;
-		border-bottom-color: var(--mat-table-row-item-outline-color, rgba(0, 0, 0, 0.12));
-		border-bottom-width: var(--mat-table-row-item-outline-width, 1px);
-		border-bottom-style: solid;
+		grid-gap: 2px 12px;
```

Stylelint (17.12.0)

[error] Expected "grid-gap" to be "gap" (property-no-deprecated)

(property-no-deprecated)